lib/checkexceptionsafety.cpp

```diff
@@ -208,8 +208,10 @@ static const Token * functionThrowsRecursive(const Function * function, std::set
     for (const Token *tok = function->functionScope->bodyStart->next();
          tok != function->functionScope->bodyEnd; tok = tok->next()) {
         if (tok->str() == "try") {
```

Comment:
I suggest:

        if (Token::simpleMatch(tok, "try {"))
            tok = tok->linkAt(1);  // skip till start of catch clauses